README.md

```diff
@@ -0,0 +1,34 @@
+# Movie Recommendation system application using Taipy
+- In the application, the user's information is secured as `user_id`.
+- The app recommends **top_k** movies for a user via user_id, also give the **top_k actual** movies that the user is interested in.
+- There are 2 models used to make recommendations: *kNN* and *MF*
+- There are two metrics to evaluate the performance of the model: *recall* and *precision*. These are calculated automatically after making recommendations
+
+## Run the application on a local machine
+- Step 1: Clone this repo to the computer.
+At the conmmand prompt, type <br/>
+<code>git clone https://github.com/caotuannghia20/recsys-taipy-app.git </code>
+ - Step 2: Download the dataset [here](https://grouplens.org/datasets/movielens/100k/)
+ - Step 3: Split file `u.data` into training (**rating_train.csv**) and test (**rating_test.csv**) data with the ratio 80%/20% using file `ultis/splitdata.py`. Edit the path in line 17
+ - Step 4: Edit the dataset path in `config/config.py` line 26, 34, 41
```

Comment:
```suggestion
 - Step 4: Edit the dataset path in `config/config.py` line 26, 34, 41
```
The path should be a constant, or at least a variable at the start of the file so that the user can easily find and edit.